retrodep/vendored.go

```diff
@@ -155,27 +155,27 @@ func updateHashesAfterStrip(hashes *FileHashes, wt WorkingTree, ref string, path
 		}
 
 		// Re-hash the altered file
-		h, err := hashes.h.Hash(path, f.Name())
+		h, err := wt.Hash(path, f.Name())
 		if err != nil {
 			return anyChanged, err
 		}
-		hashes.hashes[path] = h
+		hashes[path] = h
 		anyChanged = true
 
 	}
 
 	return anyChanged, nil
 }
 
-func matchFromRefs(strip bool, hashes *FileHashes, wt WorkingTree, subPath string, refs []string) ([]string, error) {
+func matchFromRefs(strip bool, hashes FileHashes, wt WorkingTree, subPath string, refs []string) ([]string, error) {
 	var paths []string
 	if strip {
-		for path := range hashes.hashes {
+		for path := range hashes {
```

Comment:
if it's a list of hashes should be hash the item inside, not a path.

Reply:
It's a map of hashes indexed by paths.